src/net.h

```diff
@@ -703,6 +705,68 @@ class V1TransportDeserializer final : public TransportDeserializer
     CNetMessage GetMessage(const CMessageHeader::MessageStartChars& message_start, int64_t time) override;
 };
 
+// ChaCha20 must never reuse a {key, nonce} for encryption nor may it be
+// used to encrypt more than 2^70 bytes under the same {key, nonce}
+// Re-key after 1GB (RFC4253 / SSH recommendation) or after 1h
+static constexpr unsigned int REKEY_LIMIT_BYTES = (1024 * 1024 * 1024);
+static constexpr unsigned int REKEY_LIMIT_TIME = 3600;
+static constexpr unsigned int REKEY_ABORT_LIMIT_BYTES = REKEY_LIMIT_BYTES * 1.1; // abort after ~10% tolerance buffer
+static constexpr unsigned int REKEY_ABORT_LIMIT_TIME = REKEY_LIMIT_BYTES * 1.1;  // abort after ~10% tolerance buffer
+static constexpr unsigned int MIN_REKEY_TIME = 10;                               // minimal rekey time to avoid DOS
```

Comment:
`MIN_REKEY_TIME` is unused now. Is it added intended on purpose in the future?

Reply:
Thanks. I added the check for not violating the `MIN_REKEY_TIME`.